kolla/node_custom_config/neutron/ml2_conf.ini

```diff
@@ -66,10 +66,15 @@ token = {{ switch.auth.token }}
 {% else %}
 ip = {{ switch.address }}
 username = {{ switch.auth.username }}
+{% if switch.auth.password is defined %}
 password = {{ switch.auth.password }}
+{% endif %}
 {% if switch.auth.secret is defined %}
 secret = {{ switch.auth.secret }}
 {% endif %}
+{% if (switch.auth.private_key is defined) and (switch.ngs_config.key_file is not defined) %}
```

Comment:
I think I'll merge as is, due to two factors:
1. the password can still be useful in combination, as above
2. the `key_file` gets passed in via the default ngs kwargs, so it needs to be handled separately to avoid a double template.

Reply:
Yep, sorry, that's fine :)